src/styles/_catalog.scss

```diff
@@ -1,22 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: $catalog-gap;
+  justify-content: center;
+  grid-template-columns: repeat(1, $card-width);
 
-  max-width: 944px;
-  margin: 0 auto;
+  @each $width, $col-number in $breakpoints {
+    @media (min-width: $width) {
+      grid-template-columns: repeat($col-number, $card-width);
+    }
+  }
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
```

Comment:
`width: 100%` would be better here, coz your card is always inside your grid cell.

Reply:
Oww thanks that's very very good advice:fire: